tools/release.sh

```diff
@@ -167,7 +167,8 @@ cp -v -t "$RELEASE_DIR" LICENSE NOTICE THIRD-PARTY
 check_swagger_artifact src/api_server/swagger/firecracker.yaml "$VERSION"
 cp -v src/api_server/swagger/firecracker.yaml "$RELEASE_DIR/firecracker_spec-$VERSION.yaml"
 
-cp -v test_results/test-report.json "$RELEASE_DIR/"
+cp -v test_results/test-report-functional+security.json "$RELEASE_DIR/"
```

Comment:
Can you also change https://github.com/firecracker-microvm/firecracker/blob/main/.buildkite/pipeline_pr.py#L52 to not fake the test-results.json?